src/busco/config.vsh.yaml

```
@@ -0,0 +1,206 @@
+            Parameters should be contained within a single string, without whitespace and seperated by commas.
+          example: "--PARAM1=VALUE1,--PARAM2=VALUE2"
+        - name: --augustus_species
+          type: string
+          required: false
+          description: |
+            Specify the augustus species
+        - name: --long
+          type: boolean_true
+          description: |
+            Optimize Augustus self-training mode. This adds considerably to the run time, but can improve results for some non-model organisms.
+
+    - name: BBTools Settings
+      arguments:
+        - name: --contig_break
+          type: integer
+          default: 10
+          description: |
+            Number of contiguous Ns to signify a break between contigs in BBTools analysis.
+        - name: --limit
+          type: integer
+          default: 3
+          description: |
+            Number of candidate regions (contig or transcript) from the BLAST output to consider per BUSCO.
+            This option is only effective in pipelines using BLAST, i.e. the genome pipeline (see --augustus) or the prokaryota transcriptome pipeline.
+        - name: --scaffold_composition
+          type: boolean_true
+          description: |
+            Writes ACGTN content per scaffold to a file scaffold_composition.txt.
+
+    - name: BLAST Settings
+      arguments:
+        - name: --e_value
+          type: double
+          default: 0.001
+          description: |
+            E-value cutoff for BLAST searches.
+
+    - name: Protein Gene Prediction settings
+      arguments:
+        - name: --miniprot
+          type: boolean_true
+          description: |
+            Use Miniprot gene predictor.
+
+    - name: MetaEuk Settings
+      arguments:
+        - name: --metaeuk_parameters
+          type: string
+          description: |
+            Pass additional arguments to Metaeuk for the first run (see Metaeuk documentation https://github.com/soedinglab/metaeuk).
+            All parameters should be contained within a single string with no white space, with each parameter separated by a comma.
+          example: "--max-overlap=15,--min-exon-aa=15"
+        - name: --metaeuk_rerun_parameters
+          type: string
+          description: |
+            Pass additional arguments to Metaeuk for the second run (see Metaeuk documentation https://github.com/soedinglab/metaeuk).
+            All parameters should be contained within a single string with no white space, with each parameter separated by a comma.
+          example: "--max-overlap=15,--min-exon-aa=15"
+
+  resources:
+    - type: bash_script
+      path: script.sh
+  test_resources:
+    - type: bash_script
+      path: test.sh
+    - type: file
+      path: test_data
+platforms:
+  - type: docker
+    image: ezlabgva/busco:v5.6.1_cv1
```

I propose restricting the number of "vetted" docker repositories.

Personally, I'd limit the base images to anything that's officially supported on docker hub, nvcr.io, rocker, and  quay.io/biocontainers.

In this case, I propose using

```suggestion
    image: quay.io/biocontainers/busco:5.6.1--pyhdfd78af_0
```